deploy/docker-ephemeral/docker-compose.yaml

```diff
@@ -20,6 +20,11 @@ services:
       - DEFAULT_REGION=eu-west-1
       - SERVICES=ses,sns
 
+  fake_s3:
+      image: tiagoloureiro/fake-s3:0.2.5
+      ports:
+        - 4570:4570
+
```

Comment:
There's an issue with using organisations when creating automated builds and not giving dockerhub unrestricted access (see [this](https://forums.docker.com/t/organization-cant-set-up-automated-build/2987/2) and [this](https://forums.docker.com/t/cant-access-new-github-organization-for-automated-builds/1080/17)) which is why I opted to leave this under `tiagoloureiro`